### DISCUSSION TITLE
Add optional week display for qDate

### DISCUSSION THREAD

**Discussion by chaosKP** [quasarframework/quasar#12149] [Ideas / Proposals]
Yes, I know there was already a [feature request](https://github.com/quasarframework/quasar/issues/10618) for this with the conclusion to use qCalendar, but I don't think qCalendar is a viable option (at least for use) because it has a totally different look and feel then the native qDate (no year/month selection and labeling etc).

The usability is a lot worse than qDate and only using qCalendar for those which need the week display totaly breaks the consistency of the application.

**Comment by HenrikClausen** [comment]
I absolutely agree with this.
The QCalendar has a different look and you cant set eg. font-size with the theming options.

**Comment by mik3ybark3r** [comment]
I agree with that as well. 
QDate fits our needs much better and the only missed feature is week numbers.
Are there any reasons to prevent this feature from QDate?
Or can we add a new issue for that?
